src/main/resources/pdiresources/program-workflow-state/preparation.ktr

```diff
@@ -508,10 +775,7 @@
     <single_statement>N</single_statement>
     <replace_variables>N</replace_variables>
     <quoteString>N</quoteString>
-    <sql>ALTER TABLE patient_state ADD column updated INT(1) DEFAULT 0;
-ALTER TABLE concept_state_conversion ADD column updated INT(1) DEFAULT 0;
-CREATE TABLE program_workflow_state_backup LIKE program_workflow_state; 
-INSERT program_workflow_state_backup SELECT * FROM program_workflow_state;</sql>
+    <sql>ALTER TABLE concept_state_conversion ADD column updated BOOLEAN DEFAULT 0;</sql>
```

Comment:
Here I  think  `updated` column  should have  a specific  name as   `program -workflow`  and   `concept` also have  their references   in `concept_state_conversion`